PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+Please employ to following template as the base structure of Pull Requests for **Alloy-Editor** project. Pull Requests not incorporating the sections below will be reqarded as unsatisfying and rejected. 
```

Comment:
Project name should be `AlloyEditor` (or `liferay/alloy-editor` if we want to refer to the github repo).

Possible typo `re**q**arded` -> `re**g**arded`